src/main/java/core/basesyntax/shape/Rectangle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.shape;
+
+import core.basesyntax.type.Color;
+
+public class Rectangle extends Shape {
+    private double width;
+    private double height;
+
+    public Rectangle(Color color, double width, double height) {
+        super(color);
+        this.width = width;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return this.width * this.height;
+    }
+
+    @Override
```

Comment:
Consider using an interface for drawable objects to separate the drawing functionality from the shape hierarchy.